test/sql/updates/post.compression.sql

```diff
@@ -17,7 +17,7 @@ WHERE
   hypertable.table_name = 'compress'
   AND chunk.compressed_chunk_id IS NULL;
 
-SELECT * FROM compress ORDER BY time DESC, small_cardinality;
+SELECT * FROM compress ORDER BY time DESC, small_cardinality, large_cardinality, some_double, some_int, some_custom, some_bool;
```

Comment:
We have to use the same ordering as the existing up/downgrade tests to generate the same output. Otherwise, the tests will fail. Therefore, we need an ordering of `time DESC` and `large_cardinality ASC`. I think that is not possible with the proposed change.